spec/controllers/users_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET /users' do
+    before do
+      FactoryBot.create :takashi
+      FactoryBot.create :satoshi
+    end
+
+    it 'リクエストが成功し、ユーザー名が表示されること' do
+      aggregate_failures do
+        get users_path
+        expect(response.status).to eq 200
+        expect(response.body).to include 'takashi@example.com'
+        expect(response.body).to include 'satoshi@example.com'
+      end
+    end
+  end
+
+  describe 'GET /users/:id' do
+    context 'ユーザーが存在する場合' do
+      let(:takashi) { FactoryBot.create :takashi }
```

Comment:
Idiomatic

具体的な人の名前をつけるのではなくて、
userのような一般的な変数名にしよう
```suggestion
      let(:user) { create(:user) }
```

Reply:
下記コミットで修正しました。
cad6dfad780936de81270bfcdf5d40844080dec3